addons/udes_stock/models/stock_picking_type.py

```diff
@@ -47,8 +53,8 @@ class StockPickingType(models.Model):
     u_multi_users_enabled = fields.Boolean(
         string="Multi Users Enabled",
         help="Flag to enable multi users processing same picking simultaneously. "
-             "On validate of picking will be confirmed only what is done by the specific user "
-             "unless all moves have been picked from other users on the validation time.",
+        "On validate of picking will be confirmed only what is done by the specific user "
```

Comment:
This isn't very clear.  Perhaps it should read:

> Flag to enable multiple users to work on a picking simultaneously.  On validation of a picking only the moves done by the specific user will be confirmed unless all moves have already been picked by other users.

(I think that's what it's supposed to mean...)